code/modules/projectiles/gun.dm

```diff
@@ -287,6 +287,7 @@
 		process_accuracy(projectile, user, target, accuracy_mod, disp)
 
 		if (pointblank)
+			visible_message("This should be hitting")
```

Comment:
Why this?

Reply:
This is left over from testing shouldn't be here I'll remove it